common/src/main/java/org/opensearch/sdk/UpdateDataObjectResponse.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.sdk;
+
+import org.opensearch.action.support.replication.ReplicationResponse.ShardInfo;
+import org.opensearch.core.common.Strings;
+import org.opensearch.core.index.shard.ShardId;
+
+public class UpdateDataObjectResponse {
+    private final String id;
+    private final ShardId shardId;
+    private final ShardInfo shardInfo;
```

Comment:
No, it can not be null, you have asked this same question before and it was answered here: https://github.com/opensearch-project/ml-commons/pull/2459#discussion_r1607304431